identityClient/identityClient_test.go

```diff
@@ -40,7 +41,39 @@ var (
 	accountServiceClient           = accountClient.New(ValidAccountClientConfig)
 )
 
-func createIdentityAndRegisterWithRealm(t *testing.T, realm *Realm, registrationToken string) *RegisterIdentityResponse {
+func createFakePasswordNoteForIdentity(clientConfig e3dbClients.ClientConfig, recipientKey, username, realmName string) (*storageClient.Note, error) {
+	noteName, err := e3dbClients.DeriveIdentityCredentialsNoteName(username, realmName)
+	data := map[string]string{
+		"key1": "rawnote",
+		"key2": "unprocessednote",
+		"key3": "organicnote",
+	}
+	if err != nil {
+		return nil, err
+	}
+	ak := e3dbClients.RandomSymmetricKey()
+	eak, err := e3dbClients.EncryptAccessKey(ak, clientConfig.EncryptionKeys)
+	if err != nil {
+		return nil, err
+	}
+	encryptedData := e3dbClients.EncryptData(data, ak)
+	return &storageClient.Note{
+		Mode:                "Sodium",
+		IDString:            noteName,
+		ClientID:            clientConfig.ClientID,
+		RecipientSigningKey: recipientKey,
+		WriterSigningKey:    clientConfig.SigningKeys.Public.Material,
+		WriterEncryptionKey: clientConfig.EncryptionKeys.Public.Material,
+		EncryptedAccessKey:  eak,
+		Type:                "Integration Test",
+		Data:                *encryptedData,
+		Plain:               map[string]string{"extra1": "not encrypted", "extra2": "more plain data"},
+		Signature:           "signed",
+		MaxViews:            5,
+	}, err
+}
+
+func createIdentityAndRegisterWithRealm(t *testing.T, realm *Realm, registrationToken string, clientConfig e3dbClients.ClientConfig) *RegisterIdentityResponse {
 	identityName := uuid.New().String()
 	identityEmail := identityName + "@email.com"
```

Comment:
should this be updated to a tozny email?

Reply:
I don't think these are testing any email functionality, but it's definitely a good idea to get them updated just in case. I will update them 👍

Reply:
Actually turns out these were updated, my branch was just outdated